### PR TITLE
news: mention libressl update

### PR DESCRIPTION
**Pull request by Gottox**

### CODE REVIEW

File: _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+layout: post
+title: "LibreSSL 2.3 has landed!"
+comments: true
+---
+
+We're proud to announce that yet again we are the first and currently only Linux
+distribution that ships [LibreSSL-2.3](http://www.libressl.org/) by default.
+
+In the last two days we rebuilt the whole package index with the latest stable
+release of LibreSSL. The process took more than 24 hours and needed several
```
Comment (Vaelatern) [inline _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown]: `we rebuilt the whole repository with LibreSSL-2.3` maybe?

File: _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+fixes to various packages. There are still some packages, that link against
+the older LibreSSL-2.2 but the will go away within a few days.
```
Comment (Vaelatern) [inline _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown]: `s/go away/be fixed to use the latest LibreSSL/` perhaps?

File: _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+title: "LibreSSL 2.3 has landed!"
+comments: true
+---
+
+We're proud to announce that yet again we are the first and currently only Linux
+distribution that ships [LibreSSL-2.3](http://www.libressl.org/) by default.
+
+In the last two days we rebuilt the whole repository with LibreSSL-2.3. The
+process took more than 24 hours and needed several fixes to various packages.
+There are still some packages that link against the older LibreSSL-2.2 but the
```
Comment (Vaelatern) [inline _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown]: `s/but the/but they/`

File: _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+layout: post
+title: "LibreSSL 2.3 has landed!"
+comments: true
+---
+
+We're proud to announce that yet again we are the first and currently only Linux
+distribution that ships [LibreSSL-2.3](http://www.libressl.org/) by default.
+
+In the last two days we rebuilt the whole repository with LibreSSL-2.3. The
+process took more than 24 hours and needed several fixes to various packages.
```
Comment (Vaelatern) [inline _posts/2016-03-31-LibreSSL-2.3-has-landed.markdown]: `and involved several fixes to packages that were not initially LibreSSL 2.3 compatible` to remove the "several fixes to various" bit.
Then it all looks great to me.

**Comment by Gottox** [comment]
Thanks for your help!

**Pull request merged**